### ISSUE TITLE
Backup name should be visible when editing configuration

### ISSUE THREAD

**Issue by loufidelity** [duplicati/duplicati#5025]
- [x] I have searched open and closed issues for duplicates.
- [x] I have searched the [forum](https://forum.duplicati.com) for related topics.

----------------------------------------

## Environment info
- **Duplicati version**: 2.0.7.1
- **Operating system**: I believe it does not matter
- **Backend**: Does not matter

## Description
The name of the backup that is being edited should be visible when editing a backup configuration. Usually the name of another backup is visible on top in the status bar, and this causes frequent confusion for me.

## Steps to reproduce
1. Edit backup configuration
2. The name of the backup being edited is not visible

- **Actual result**:
The name of the backup being edited is not visible
- **Expected result**:
The name of the backup being edited should be visible

## Screenshots
![duplicati backup title](https://github.com/duplicati/duplicati/assets/66403681/61717634-93f6-43d1-8a2f-28fad0d8787d)

**Comment by ts678** [comment]
Opinions vary, and maybe we'll get some others, but "it looks like I am editing backup 1" could be fixed by reading context.
Seeing "Next scheduled task" before the job name, and a time after it should eventually reduce confusion on current status.
If you ever forget which job you're editing, you could click to "1 General" screen to remind yourself of the `Name` of this job.

[Components of the Graphical User Interface](https://duplicati.readthedocs.io/en/latest/03-using-the-graphical-user-interface/#components-of-the-graphical-user-interface) in the Duplicati User's Manual puts it as follows:

> The Status Bar shows information about the currently running Backup or Restore job. If no operation is active, the next scheduled backup job is showed here. If there are no scheduled backup jobs, the Status Bar shows "No scheduled tasks".

and I'm not sure that adding an "except when editing" is a good change, even if possible, and I'm not certain of its difficulty.
You can be sitting there editing when a scheduled backup fires up. At that point, you might like status bar to work as usual.

One might say there should be another job name, for example above the navigation bar, and that argument sounds better.
Regardless, action (or not) will depend on whether the developers see this as a priority. Lots of asks and too few volunteers.

**Comment by darrelfrancis** [comment]
Yes, it's probably an exaggeration to say "it looks like I am editing backup 1".

However the @loufidelity is right, it would indeed help if the title of the main part of the page was changed from
"Scheduling" to "Scheduling backup 2"

**Comment by ts678** [comment]
> title of the main part of the page was changed from "Scheduling" to "Scheduling backup 2"

Presumably that meant "Schedule". Something like that seems helpful, but as usual there are lots of other requests in the queue.

The grammar here might have to be checked (somehow) for what the translators have done. Maybe it'd deserve "Schedule for"...
Another thing to check is how a long name in a huge font fares. Backups actually have numbers, but I assume you'd want name.
Counter-argument (or further extension) would be to change every one of the 5 screens the same way, as all have a similar issue except maybe for the "General backup settings" screen where having it extended with the name right above `Name` is redundant.

I've marked this as an "enhancement", and discussion can continue. Possibly at some point some volunteer will decide to code it, however there is more developer priority now on things like backup reliability and performance. This one seems a "nice to have".

**Comment by loufidelity** [comment]
I will make it more clear what I had in mind.

It would have two headers.

First header: 
Backup name (what I refer to as "backup 2" but would be the name of the backup you are editing).
If you have changed the backup name in the General section but not yet saved it could be still showing the old backup name. Or some default name "New backup" if it does not yet have a name. 
Or maybe it could even show the new backup name that has not been saved yet. This is up for discussion but either solution would make it clear to the user what they are editing.

Second header: Schedule 
...or General, Destination, etc...

This is what I was suggesting but in an indirect way.

Sorry to take so long to comment. I appreciate the responses!

**Comment by jg3** [comment]
+1

I noticed this same thing tonight while creating a backup.  I wanted to label the path inside the S3 bucket to be the same as the job name for consistency.  It seems like a small thing but whilst editing a job, the job being edited should be clearly labeled on each screen.

**Comment by 5-u5** [comment]
Since both adding and editing a backup direct to the same page (Duplicati/Server/webroot/ngax/templates/addoredit.html), they might need to be split into two files or, if there is some way to tell which action the user is performing, only display the header when a backup is being edited.

However if displaying the name of the backup even during its creation is preferred, I followed the format of a previous comment

> First header:
Backup name...

![image](https://github.com/duplicati/duplicati/assets/117954449/3b7f1867-78ba-4793-814e-c965f198ebdc)
by adding (Duplicati/Server/webroot/ngax/templates/addoredit.html:33)
`<h1 ng-show="CurrentStep > 0" ng-hide="CurrentStep == 0">{{'Backup name' | translate}}: {{Backup.Name}}</h1>`
And added (Duplicati/Server/webroot/ngax/templates/addoredit.html:35) for consistency:
`<h1>{{'Add/Edit Backup' | translate}}</h1>`
![image](https://github.com/duplicati/duplicati/assets/117954449/f9fb1095-e327-43dd-a499-a28e2757c215)
But once again the issue about both adding and editing being on the same page pops up, and I think it's not perfect for the header to always say Add/Edit as it might be confusing, so does anyone have any suggestions?

**Comment by kenkendk** [comment]
Fixed with new UI
